models/dim_dbt__current_models.sql

```diff
@@ -0,0 +1,63 @@
+with base as (
+    select *
+    from {{ ref('stg_dbt__models') }}
+),
+
+model_executions as (
+    select *
+    from {{ ref('stg_dbt__model_executions') }}
+),
+
+latest_models as (
+    /* Retrieves the models present in the most recent run */
+    select *
+    from base
+    where run_started_at = (select max(run_started_at) from base)
+),
+
+latest_models_runs as (
+    /* Retreives all successful run information for the models present in the most
+    recent run and ranks them based on query completion time */
+    select
+        model_executions.node_id
+        , model_executions.was_full_refresh
+        , model_executions.query_completed_at
+        , model_executions.total_node_runtime
+        , model_executions.rows_affected
+        , row_number() over (
+            partition by latest_models.node_id, model_executions.was_full_refresh
+            order by model_executions.query_completed_at desc /* most recent ranked first */
+        ) as run_idx
+    from model_executions
+    inner join latest_models on model_executions.node_id = latest_models.node_id
+    where model_executions.status = 'success'
+),
+
+latest_model_stats as (
+    select
+        node_id
+        , case when was_full_refresh then query_completed_at end as last_full_run_completed_at
+        , case when was_full_refresh then total_node_runtime end as last_full_run_total_runtime
+        , case when was_full_refresh then rows_affected end as last_full_run_rows_affected
```

Comment:
Let's call these `last_full_refresh_x` vs `last_full_run_x`, realise it's a schema but it's more explicit